01week/datatypes.js

```diff
@@ -0,0 +1,55 @@
+// 1. Write a JavaScript program to display the current day and time.
+const todaysDate = new Date();
+console.log(todaysDate);
+
+
+// 2. Write a JavaScript program to convert a number to a string.
+const numberConvert = 500;
```

Comment:
this should be a function that takes in any number and returns the number as a string